controllers/accounts.js

```diff
@@ -66,6 +66,7 @@ router.post('/token', async (req, res) => {
     }
 
     console.log(req.body);
+    console.log(req.dne.asdf.s.df.sdf.dsf);
```

Comment:
What is this?

Reply:
this was a test line, i forgot to delete it